wordpress-mobile/WordPress-Android
# Show visitors over views in the overview usecase

Fixes #9028 
This PR adds adds the visitors as an overlay to the Overview block. Now when the user has `Views` selected, there is an overlapping bar in dark blue color that represents the visitors. We've also added a legend that explains it. The difference is visible even when highlighted. The legend is not visible when other types (Visitors/Likes/Comments) are selected.

To test:
* Open Stats - Days/Weeks/Months/Years
* Select Views as the type in the Overview block
* There are a dark overlapping bars that display visitors over views
* There is a legend visible that explains the dark bars
* Select Visitors/Likes/Comments
* Notice that the dark bars are gone
* Notice that the legend is also gone

![screenshot_1550506126](https://user-images.githubusercontent.com/1079756/52963380-e79aa880-339f-11e9-9e52-ad96e59293ca.png)

Update release notes:
